modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -136,9 +136,13 @@ def build_target_veteran(veteran_id:, loa:)
         target_veteran.mpi_record?(user_key: veteran_id)
         mpi_profile = target_veteran.mpi.mvi_response.profile
 
+        target_veteran[:first_name] = mpi_profile[:given_names].first # NOT ideal...is this guaranteed?
```

Comment:
@bosawt 
Do you know how reliable this `given_names` attribute is?
Like, are we "safe" to assume that it'll always have at least a single value?
Or maybe there's a better way for us to retrieve a "first name" value from the MPI/MVI response?

Reply:
If the name exists on the response, `given_names.first` will always be the first name. I do see instances where the MPI response won't have a name (which shouldn't happen, so these cases might be a data error on MPI's end), so it might be best to put a safe accessor here or to raise an error if `given_names` is `nil`

Reply:
@bosawt 
Thx for the response!
Just to clarify, are you saying that in the rare case of a possible data error in MPI, the `given_names` attribute could be `nil` (rather than an empty array).  So we should protect ourselves by doing something like, 
`mpi_profile[:given_names]&.first`?

Reply:
Yeah, if we raise an error when parsing first or last name in the MPI response itself, currently we log a message to and return `given_names: nil, family_name: nil`